Makefile

```diff
@@ -26,3 +26,7 @@ unit-test:
 lint:
 	$(CONTAINER_ENGINE) run --rm -v $(PWD):$(PROJECT_WORKING_DIR):Z --workdir $(PROJECT_WORKING_DIR) $(LINTER_IMAGE_NAME):$(LINTER_IMAGE_TAG) golangci-lint run -v
 .PHONY: lint
+
+push: build
+	$(CONTAINER_ENGINE) push $(CHECKUP_IMAGE_NAME):$(CHECKUP_IMAGE_TAG)
```

Comment:
I think it is worth enabling overriding the base image name (line 3 `:=` -> `?=`)  to enable using this target to push the image to your own/local registry.

Reply:
Done.